PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json

```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "a6b35002-048d-4c78-8a39-85ddb1a7a41f",
```

Comment:
Best practice is to exclude the update of this id. It's changed every time you save it.

Reply:
Yes, updated